components/Search/Form.js

```diff
@@ -37,14 +37,18 @@ const Form = compose(
     t,
     searchQuery,
     setSearchQuery,
-    onSearchSubmit
+    onSearchSubmit,
+    noInitialFocus
   }) => {
     const [focusRef, setFocusRef] = useState(null)
     const [formValue, setFormValue] = useState(urlQuery)
     const [slowFormValue] = useDebounce(formValue, 200)
-
     useEffect(() => {
-      startState && focusRef && focusRef.input && focusRef.input.focus()
+      startState &&
+        !noInitialFocus &&
+        focusRef &&
+        focusRef.input &&
+        focusRef.input.focus()
     }, [startState, focusRef])
```

Comment:
```suggestion
    }, [startState, focusRef, noInitialFocus])
```
(`react-hooks/exhaustive-deps` – let's add it to our eslint setup soon)